@lingosta/app/components/Group/LanguageSelection.tsx

```diff
@@ -11,7 +11,9 @@ interface LanguageSelectionProps {
 }
 
 const LanguageSelection = ({ setLanguage }: LanguageSelectionProps) => {
-  const languages = SUPPORTED_LANGUAGES.map((language: Language) => {
+  const languages = SUPPORTED_LANGUAGES
+  .filter((language: Language) => getLanguageName(language) !== 'Unknown')
```

Comment:
So there's a style of coding where people literally just write comments about what the code is doing. Sometimes it gets very excessive and I don't like it.

I think this is a perfect opportunity to write a comment to explain the _why_ behind this change. That way, a React expert who has no context about our application can understand why it's done this way.

Could you add this?
```
// Since we want users to select a valid language from the list, we filter out the "Unknown" language which is used as a catch-all in `getLanguageName`
```